### DISCUSSION TITLE
WindowsAppSDK 1.1.2 released?

### DISCUSSION THREAD

**Discussion by Marv51** [microsoft/WindowsAppSDK#2697] [Dev team discussions]
You half-released the 1.1.2 update on a Friday evening before a holiday weekend.

I'm guessing the nuget package was accidentally published early? 
Release notes, the git tag, tweets, announcement issue etc. are still missing.

**Comment by ghost1372** [comment]
https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/stable-channel#version-112
